pom.xml

```diff
@@ -65,7 +65,7 @@ under the License.
         <flink.version>1.11.1</flink.version>
         <debezium.version>1.2.1.Final</debezium.version>
         <testcontainers.version>1.14.3</testcontainers.version>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

Comment:
I think we don't need to upgrade java version. Compiling in java 8 should be compatible with java 11.

Reply:
Ok, I will rollback this change.

Reply:
done